Project_Proposal_Flood.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "Multispectral and Hyperspectral comparison in South Africa's Greater Cape Floristic Region"
+author: Willow Flood
```

Comment:
This all looks great.  You will also need to account for vegetation age, but we can chat about that later.